internal/pkg/deploy/upload/asset/asset.go

```diff
@@ -140,16 +143,20 @@ func (u *ArtifactBucketUploader) uploadAssets(assets []asset) error {
 	return g.Wait()
 }
 
-// uploadAssetMappingFile uploads a JSON file to u.AssetMappingPath containing
-// the current location of each file in the artifact bucket and the desired location
+// uploadAssetMappingFile uploads a JSON file containing the current
```

Comment:
nit: why the "current" distinction, can the objects move in the artifact bucket?

Reply:
ah i see that might be a little confusing, i could probably just drop the word. i meant "location of the file _right now_" as opposed to the service bucket location